Added gCreate gMax printer profiles

gordo3di opened the pull request:
We have the profiles completed for the gMax series of printers including the 3d models. We have been testing the profiles for a while using PrusaSlicer 2.3.1 and they have been performing very well. Please let me know if we have to make any changes. Thank you!

rtyr commented:
Thank you. I will check it.

rtyr commented:
Your profiles are now merged manually on master (it will be included in PrusaSlicer 2.4.0-alpha).

I fixed/changed some things:

- Issues in the ini file (some invalid inheritance, unnecessary `renamed_from` keys, `travel_speed` in printer settings, etc.).

- Top solid infill extrusion width was set to 90% and 150% in some profiles. If expressed as a percentage, it will be computed over layer height (not over nozzle diameter). For example, 90% produced 0.36mm extrusion width with 1mm nozzle, so it wouldn't be even possible to slice anything. I changed it to 0 for now.

- Reworked bed models. I fixed some errors and reduced bed models only to build platform. Complete printer model may look cool at the beginning, but it is not very practical (it can obstruct a view while rotating, etc.). We are considering adding a new option, which would allow to load printer model separately from the bed model.

- Reworked thumbnails.

Closing.

gordo3di commented:
> Your profiles are now merged manually on master (it will be included in PrusaSlicer 2.4.0-alpha).
> 
> I fixed/changed some things:
> 
> * Issues in the ini file (some invalid inheritance, unnecessary `renamed_from` keys, `travel_speed` in printer settings, etc.).
> * Top solid infill extrusion width was set to 90% and 150% in some profiles. If expressed as a percentage, it will be computed over layer height (not over nozzle diameter). For example, 90% produced 0.36mm extrusion width with 1mm nozzle, so it wouldn't be even possible to slice anything. I changed it to 0 for now.
> * Reworked bed models. I fixed some errors and reduced bed models only to build platform. Complete printer model may look cool at the beginning, but it is not very practical (it can obstruct a view while rotating, etc.). We are considering adding a new option, which would allow to load printer model separately from the bed model.
> * Reworked thumbnails.
> 
> Closing.

Perfect thank you. I'll keep those settings in mind re the top solid infill extrusion width. I actually like the 0 better and I think I just forgot to change it in a few of the profiles. Also for the 3d models I figured they might be overkill so thanks for reworking them.

In the future if we want to release updated settings should I push them through the github again? I ask because I saw that it looks like the profiles pull from your servers for updates and not this repository.

https://files.prusa3d.com/wp-content/uploads/repository/PrusaSlicer-settings-master/live/gCreate/

I can send you my email if you want. I'm one of the owners of gCreate.

Thanks again for your help!

rtyr commented:
> In the future if we want to release updated settings should I push them through the github again? 

You can create pull request here again or at prusa3d/PrusaSlicer-settings.

> I ask because I saw that it looks like the profiles pull from your servers for updates and not this repository.

Our server is automatically synchronized with https://github.com/prusa3d/PrusaSlicer-settings.
